src/layers/MapboxLayer.js

```diff
@@ -58,7 +58,7 @@ export default class MapboxLayer extends Layer {
 
         // adjust view parameters in mapbox
         const { rotation } = viewState;
-        if (rotation && this.renderState.rotation !== rotation) {
+        if (rotation != null && this.renderState.rotation !== rotation) {
```

Comment:
it could potentially break if rotation is undefined .
```suggestion
        if (this.renderState.rotation !== rotation) {
```

Reply:
In JS: null == undefined

Reply:
you're right but it's weird eslint didn' complain about this. Normally wit should accept only !==